### DISCUSSION TITLE
Is there an available JSON schema for `settings.json`?

### DISCUSSION THREAD

**Discussion by justDeeevin** [zed-industries/zed#12540] [Feature Requests]
I can't seem to find one, but I've seen some things indicating that it probably exists.

**Comment by dudeofawesome** [comment]
I found [this file](https://github.com/zed-industries/zed/blob/main/crates/settings/src/json_schema.rs) which seems to maybe be involved in the generation of the JSON schema you're looking for.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
